rocky/rocky/templates/partials/form/field_input_radio.html

```diff
@@ -1,24 +1,25 @@
-<div>
-    {{ field.label_tag }}
+<fieldset>
+  <legend>
+    {{ field.label }}
```

Comment:
If I remember correctly, it's just the name field.label right? As it actually contains the content of the legend but can't be called legend because of Django expectations/behaviour. If that is the case it's not ideal but ok. If we can have field.legend that would be the correct name.

Reply:
This is correct, unfortunately tunatly Django doesn't provide a native "legend" attribute to use for this AFAIK.